sql/example_patient_data.sql

```diff
@@ -1,28 +1,28 @@
 INSERT INTO `patient_data` 
-(`title`, `language`, `financial`, `fname`, `lname`, `mname`, `DOB`, `street`, `postal_code`, `city`, `state`,  `drivers_license`, `ss`, `occupation`, `phone_home`, `phone_biz`, `phone_contact`, `phone_cell`, `status`, `contact_relationship`, `date`, `sex`, `referrer`, `referrerID`, `providerID`, `email`, `ethnoracial`, `interpretter`, `migrantseasonal`, `family_size`, `monthly_income`, `homeless`, `financial_review`, `pubpid`, `pid`, `genericname1`, `genericval1`, `genericname2`, `genericval2`) VALUES ( 'Ms.', 'english', '', 'Farrah', 'Rolle', 'A.', '1973-10-11', '111 Main Street', '92101', 'San Luis', 'CA', '', '456789123', '', '(619) 555-2222', '(619) 555-3333', '(619) 555-1111', '', 'married', 'Joe', '2003-11-17 12:06:00', 'Female', 'Phil Martin', '', NULL, 'frolle@pennfirm.com', 'Latina', '', '', '3', '3200', '', '0000-00-00 00:00:00', '10', 5,'','','','');
+(`title`, `language`, `financial`, `fname`, `lname`, `mname`, `DOB`, `street`, `postal_code`, `city`, `state`,  `drivers_license`, `ss`, `occupation`, `phone_home`, `phone_biz`, `phone_contact`, `phone_cell`, `status`, `contact_relationship`, `date`, `sex`, `referrer`, `referrerID`, `providerID`, `email`, `ethnoracial`, `interpreter`, `migrantseasonal`, `family_size`, `monthly_income`, `homeless`, `financial_review`, `pubpid`, `pid`, `genericname1`, `genericval1`, `genericname2`, `genericval2`) VALUES ( 'Ms.', 'english', '', 'Farrah', 'Rolle', 'A.', '1973-10-11', '111 Main Street', '92101', 'San Luis', 'CA', '', '456789123', '', '(619) 555-2222', '(619) 555-3333', '(619) 555-1111', '', 'married', 'Joe', '2003-11-17 12:06:00', 'Female', 'Phil Martin', '', NULL, 'frolle@pennfirm.com', 'Latina', '', '', '3', '3200', '', '0000-00-00 00:00:00', '10', 5,'','','','');
```

Comment:
we are stuck with this misspelling fixed in this script and can't change this one (since it's already populated in database); notably that the label Interpreter (this is what users see) is spelled correctly, at least.